### ISSUE TITLE
How to access host port from selenoid docker browser container

### ISSUE THREAD

**Issue by valburyakov** [aerokube/selenoid#201]
Hello, i wonder if there is a possibility how to map a host port to docker container in order to access application which runs locally on my host ?  I didn't found any options how i can set --add-host option in selenoid browser docker images. Thanks in advance

**Comment by aandryashin** [comment]
Try to use host's eth0 (linux) or en0 (mac) ip address instead of 127.0.0.1 or localhost within container.

**Comment by valburyakov** [comment]
I tried to use host's ip address,  create alias `ifconfig lo0 alias 123.123.123.123/24` and even special Mac-only DNS name `docker.for.mac.localhost` but couldn't get host inside container.

**Comment by aandryashin** [comment]
I was able to access my mac's host interface from container:

```
$ ifconfig en0 | grep inet
    inet6 fe80::c9e:b043:9ddc:533b%en0 prefixlen 64 secured scopeid 0x4
    inet 192.168.1.33 netmask 0xffffff00 broadcast 192.168.1.255
```
```
$ docker run -it --rm ubuntu:16.04 /bin/bash
```
```
root@f171c702c8b7:/# ping 192.168.1.33
PING 192.168.1.33 (192.168.1.33) 56(84) bytes of data.
64 bytes from 192.168.1.33: icmp_seq=1 ttl=37 time=0.478 ms
64 bytes from 192.168.1.33: icmp_seq=2 ttl=37 time=0.184 ms
```

2017-08-30 0:21 GMT+03:00 Valentin Buryakov <notifications@github.com>:

**Comment by valburyakov** [comment]
Thx, it works and but the problem is that for auth purpose (which uses external API) i should use dns-name instead of certain IP, dev.my-company.com. So is that possible to add host for selenoid docker containers with browsers, like -add-host=dev.my-company.com:${MY_IP}

**Comment by vania-pooh** [comment]
@valburyakov currently this is not supported. And according to https://github.com/moby/moby/issues/23177 you can't specify this mapping in Docker daemon settings. Let us think a bit whether we could implement this.

**Comment by vania-pooh** [comment]
@valburyakov you can try this by using `aerokube/selenoid:latest` and specify `"hosts": ["example.com:192.168.0.1"]` in config.json.

**Comment by vania-pooh** [comment]
This works. Closing.

**Comment by valburyakov** [comment]
@vania-pooh Thx a lot for this feature. This is exactly what I needed. I appreciate it.

**Comment by vania-pooh** [comment]
@valburyakov this was released in Selenoid 1.3.6. So you can now use `aerokube/selenoid:1.3.6`.
